stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -730,6 +730,32 @@ public func withUnsafeThrowingContinuation<T>(
   }
 }
 
+// HACK: a version of withUnsafeContinuation that uses the unsafe
+// @_unsafeInheritExecutor.
```

Comment:
Would you mind adding a "why" here, it can be difficult remembering why all those `HACK:` exist sometimes 😉 

 ```suggestion
// @_unsafeInheritExecutor. The type checker will automatically replace calls to 
// #isolation-based methods to their `__abi__unsafeInheritExecutor_` version
// if called from an @_unsafeInheritExecutor context.
```

Reply:
I'm switching to this boilerplate:

```
// Note: hack to stage out @_unsafeInheritExecutor forms of various functions
// in favor of #isolation. The _unsafeInheritExecutor_ prefix is meaningful
// to the type checker.
//
// This function also doubles as an ABI-compatibility shim predating the
// introduction of #isolation.
```

Reply:
Sounds good to me, thank you!